### PR TITLE
Add a variant to Text block to support highlight text area

### LINKED ISSUE #140: Add a variant to text block to support highlight text area
image

https://www.sunstar-foundation.org/en/awards/world-hygienist/how-to-apply

### PR DESCRIPTION
**Pull request by JiangLong2019**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fixes #140 

Test URLs:
- Original: https://www.sunstar-foundation.org/en/awards/world-hygienist/how-to-apply
- Before: https://main--sunstar-foundation--hlxsites.hlx.live/en/awards/world-hygienist/how-to-apply
- After: https://quote-highlight--sunstar-foundation--hlxsites.hlx.live/en/awards/world-hygienist/how-to-apply

- [x] New variations introduced in this PR

Variation Name    | Documentation
------------- | -------------
Quote(highlight)   |

### CODE REVIEW

File: blocks/quote/quote.css [hunk 1]
```diff
@@ -47,3 +47,28 @@
   position: relative;
   font-style: normal;
 }
+
+.block.quote.highlight {
```
Comment (jindaliiita) [inline blocks/quote/quote.css:51]: @JiangLong2019 I think this is a use case of text block.
For text block we have one light background variant.
You can implement left border variant in that.
And for text "IMPORTANT" we can check with Dejan if its necessary to be in orange color

Comment (JiangLong2019) [reply blocks/quote/quote.css:51]: @jindaliiita You mean "Text Block with light background", let me check

Comment (jindaliiita) [reply blocks/quote/quote.css:51]: Yes

Comment (jindaliiita) [reply blocks/quote/quote.css:51]: We can simply add one more variant to text block i.e. left border

**Pull request closed without merging**